src/com/github/Kraken3/AFKPGC/EventHandlers.java

```diff
@@ -46,10 +46,6 @@ public void onPlayerMoveEvent(PlayerMoveEvent event) {
 		registerActivity(event.getPlayer());
 	}	
 	@EventHandler
-	public void onPlayerChatEvent(PlayerChatEvent event) {

```

Comment:
It's deprecated and the only use for it was bots keeping themselves from getting afk kicked by writing in a private groupchat (thats how almost all the non-moving bots do it). My reasoning was that anyone who actually chats will at least move a bit or trigger any other event to keep himself from being kicked. And even if he doesn't, he will still get the countdown for the kick.